<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseWindow</class>
 <widget class="QMainWindow" name="DataBaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование базы данных</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../pics/Databaseicon.png</normaloff>../../pics/Databaseicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Таблица : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Студенты</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Преподаватели</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Кафедры</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Группы студентов</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Аудитории</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Клиенты Telegram</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lView">
        <property name="text">
         <string>Группа : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addRecord"/>
   <addaction name="editRecord"/>
   <addaction name="refreshView"/>
   <addaction name="delRecord"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actTimeTable"/>
   <addaction name="act_AddClient"/>
   <addaction name="act_Browser"/>
  </widget>
  <action name="addRecord">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/Add.png</normaloff>../../../pics/Add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить запись</string>
   </property>
   <property name="toolTip">
    <string>Добавить запись</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="delRecord">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/Del.png</normaloff>../../../pics/Del.png</iconset>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
   <property name="toolTip">
    <string>Удалить запись</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="editRecord">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/Edit.png</normaloff>../../../pics/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать запись</string>
   </property>
   <property name="toolTip">
    <string>Редактировать запись</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="refreshView">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/refresh.png</normaloff>../../../pics/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить таблицу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actTimeTable">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/schedule.png</normaloff>../../../pics/schedule.png</iconset>
   </property>
   <property name="text">
    <string>Рассписание занятий</string>
   </property>
   <property name="toolTip">
    <string>Рассписание занятий</string>
   </property>
  </action>
  <action name="act_AddClient">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/addClient.png</normaloff>../../../pics/addClient.png</iconset>
   </property>
   <property name="text">
    <string>Добавить меседжер клиента</string>
   </property>
   <property name="toolTip">
    <string>Добавить меседжер клиента</string>
   </property>
  </action>
  <action name="act_Browser">
   <property name="icon">
    <iconset>
     <normaloff>../../../pics/browser.png</normaloff>../../../pics/browser.png</iconset>
   </property>
   <property name="text">
    <string>Открыть браузер</string>
   </property>
   <property name="toolTip">
    <string>Открыть браузер</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
